admin/environment-management/workspace-providers.md

```diff
@@ -4,17 +4,35 @@ description: Learn how workspace providers can minimize latency for users.
 state: beta
 ---
 
-Workspace Providers are a forthcoming feature that enables a single Coder
-deployment to provision and manage workspaces across multiple Kubernetes
-clusters and namespaces, including ones located in other geographies, regions,
-or clouds.
+Workspace Providers are logical groups of resources developers can deploy
+workspaces onto. Workspace Providers enable a single Coder deployment to
+provision and manage workspaces across multiple Kubernetes clusters and
+namespaces, including ones located in other geographies, regions, or clouds.
 
 Distributed teams can use this feature to allow users to manage workspaces in
 the cluster that's nearest, reducing network latency and improving developers'
 experience. Additionally, you can use workspace providers to support data
 sovereignty requirements or increase the isolation between workspaces running in
 the same region or cluster.
 
+### Built-In Workspace Provider
+
+By default, all Coder deployments will have a `built-in` Workspace Provider that
+identifies the kubernetes cluster the Coder deployment is in. This enables
+workspaces to be deployed into the same cluster as the Coder deployment with no
```

Comment:
Suggest using active voice, "This allows users to create workspaces in the same cluster as the Coder deployment with no additional configuration."